meson.build

```diff
@@ -23,8 +23,15 @@ config_h = configure_file(
   output: 'config.h',
 )
 
+tables_h_name = 'tables.h'
+drm_fourcc_header_path = libdrm.get_pkgconfig_variable('includedir') + '/libdrm/drm_fourcc.h'
+
+tables_h = custom_target('tables_h',
+  output : tables_h_name,
+  command : [meson.source_root() + '/fourcc.py', drm_fourcc_header_path, tables_h_name, '@OUTPUT@'])
```

Comment:
Use the meson python module to find the python installation, and call this using it.